policyengine_us/tests/policy/baseline/gov/states/al/tax/income/deductions/al_standard_deduction.yaml

```
@@ -0,0 +1,30 @@
+- name: Single filer, 6000 AGI 
```

Also name other tests similarly
```suggestion
- name: Single filer with $6,000 taxable income
```